content/en/_index.html

```diff
@@ -102,83 +94,76 @@ <h5 class="card-title text-white section-head">Model serving</h5>
         </div>
       </div>
       <div class="card border-primary-dark">
         <img
-          src="/docs/images/logos/MultiPlatform.png"
-		  class="card-img-top"
-		  draggable="false"
+          src="/docs/images/logos/tensorflow-pytorch.png"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
-          alt="Multiple Framework Logos"
+          alt="TensorFlow + PyTorch Logo"
         />
         <div class="card-body bg-primary-dark">
-          <h5 class="card-title text-white section-head">Multi-framework</h5>
+          <h5 class="card-title text-white section-head">Model Training</h5>
           <p class="card-text text-white">
-            Our development plans extend beyond TensorFlow. We're working
-			hard to extend the support of 
-			<a target="_blank" rel="noopener" href="/docs/components/training/pytorch/">PyTorch</a>, 
-			<a target="_blank" rel="noopener" href="/docs/components/training/mxnet/"> Apache MXNet</a>, 
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/mpi/"> MPI</a>,
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/xgboost/"> XGBoost</a>,
-			<a target="_blank" rel="noopener" href="/docs/components/training/chainer/"> Chainer</a>, and more. We
-			also integrate with 
-			<a target="_blank" rel="noopener" href="https://istio.io/">Istio</a> and 
-			<a target="_blank" rel="noopener" href="https://www.getambassador.io/">Ambassador</a> for ingress, 
-			<a target="_blank" rel="noopener" href="https://nuclio.io/">Nuclio</a> as a
-            fast multi-purpose serverless framework, and <a target="_blank" rel="noopener" href="https://www.pachyderm.io/">Pachyderm</a> for managing
-            your data science pipelines.
+            <a href="/docs/components/training/" target="_blank" rel="noopener" >Kubeflow Training Operators</a>
+            provide a unified, Kubernetes interface for training machine learning models on multiple frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
           </p>
         </div>
       </div>
       <div class="card border-primary-dark">
         <img
           src="/docs/images/logos/Community.png"
-		  class="card-img-top"
-		  draggable="false"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
           alt="People Icon"
         />
         <div class=" card-body bg-primary-dark">
           <h5 class="card-title text-white section-head">Community</h5>
           <p class="card-text text-white">
-            We are an open and welcoming community of software developers, data scientists, and organizations!
-            Join our <a target="_blank" rel="noopener" href="/docs/about/community/#kubeflow-slack">Slack Workspace</a>!
+            We are an open and welcoming <a href="/docs/about/community/" target="_blank" rel="noopener">community</a> of software developers, data scientists, and organizations!
```

Comment:
Should we have separate section after `Components`, and named it as `Kubeflow Ecosystem` ?
There we can add links for: Community, Kubeflow Distributions, Kubeflow External Add-Ons ?

Reply:
We should discuss this in a separate proposal, as it's less important than getting the current homepage fixed, and will probably derail the discussion.

Also, from a visual perspective, we would need to consider how not to have a multiple of 3 cards per row, if we remove one from components.

Reply:
Sounds good, we can always add another card for Central Dashboard and have Ecosystem block at the bottom.

Reply:
I have replaced the community card with "dashboard" and created a new logo for it.